includes/apple-exporter/components/class-body.php

```diff
@@ -476,12 +482,12 @@ private function dropcap_determination( $theme, $html ) {
 		/**
 		 * Allows for filtering of the dropcap content before return.
 		 *
-		 * @since 2.4.0
-		 *
 		 * @param bool                  $use_dropcap Whether to apply a dropcap to this paragraph or not.
 		 * @param string                $html The post content to filter.
-		 * @param \Apple_Exporter\Theme $theme The theme whose dropcap options are used.
+		 * @param Theme $theme The theme whose dropcap options are used.
 		 * @param string                $post_id The id of the post whose content we're parsing.
```

Comment:
do we need to align all the param names now that `\Apple_Exporter\` has been removed?